tasks/bridge/deposits.ts

```diff
@@ -37,6 +37,11 @@ task(TASK_BRIDGE_DEPOSITS, 'List deposits initiated on L1GraphTokenGateway')
       `Tracking 'DepositFinalized' events on L2GraphTokenGateway (${l2Gateway.address}) from block ${l2StartBlock} onwards`,
     )
 
+    const amount = await (
```

Comment:
did you add this additional query on purpose? if so, it might be better to store the result and use it for depositInitiatedEvents below?

But also, isn't it this redundant with what's printed on line 73?

Reply:
Fixed!